src/Pages/GlobalConfigurations/BuildInfra/utils.tsx

```diff
@@ -485,14 +474,23 @@ export const unitSelectorStyles = () =>
         }),
         option: (base, state) => ({
             ...base,
-            padding: '4px 12px',
-            backgroundColor: getOptionBackgroundColor(state.isFocused, state.isSelected),
-            color: 'var(--N900)',
```

Comment:
Pls confirm if we don't need the default color

Reply:
Not an issue